cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/RemoteValidationRequest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.datapipeline.service;
+
+/**
+ * RemoteValidationRequest
+ */
+public class RemoteValidationRequest {
+
+  private final String namespace;
+  //The original serialized request
+  private final String serializedRequest;
+
+  public RemoteValidationRequest(String namespace, String serializedRequest) {
```

Comment:
The request itself is just a string, right? Doesn't need to call it serialized.

Reply:
Done.